onebusaway-android/src/amazon/java/org/onebusaway/android/map/googlemapsv2/VehicleOverlay.java

```diff
@@ -91,25 +92,30 @@
 
     private static final int NORTH = 0;
 
-    private static final int NORTH_WEST = 1;
```

Comment:
Have you tested to make sure that these changes in numeric indexes don't result in flipped icons?  For example, `2` used to be `WEST` and now it's `EAST`.  IIRC the halfWinds generated by `MathUtils` go in counter-clockwise order, which is why this was reversed.  If that's correct, then from a quick scan of the code I believe the "WEST" direction would now result in an "EAST" icon?

If this is correct, then we definitely need better comments (i.e., I should have added better comments), as this wasn't clear in the code itself.